nncf/onnx/graph/model_transformer.py

```diff
@@ -377,6 +383,24 @@ def _apply_bias_correction_transformations(
             bias_initializer.CopyFrom(new_bias_tensor)
         return model
 
+    @staticmethod
+    def _apply_weight_update_transformations(
+        model: onnx.ModelProto, transformations: List[ONNXWeightUpdateCommand]
+    ) -> onnx.ModelProto:
+        """
+        :param model:
+        :param transformations:
+        :return:
+        """
+        name_to_node_map = get_name_to_node_map(model)
```

Comment:
Please, add a test for a new transformation

Reply:
Please take another look. The code has been reorganized, and there is no new transformation.